src/test/java/jenkins/plugins/office365connector/CardBuilderTest.java

```diff
@@ -505,4 +506,54 @@ public void getEscapedDisplayName_OnNameWithSpecialCharacters_EscapesSpecialChar
         // then
         assertThat(displayName).isEqualTo("this\\_is\\_my\\-very\\#special \\*job\\*");
     }
+
+    @Test
+    public void getCardThemeColor_OnSuccessResult_ReturnsGreen() {
+        Result successResult = Result.SUCCESS;
```

Comment:
add given/when/then to clarify what are you testing

Reply:
I was not previously familiar with given/when/then, but I believe I added them properly